section2/exercises/if-statements.js

```diff
@@ -82,15 +97,25 @@ When cupsOfFlour = 2 and hasSauce = true, your program should log "I can make pi
 When cupsOfFlour = 3 and hasSauce = true, your program should log "I can make pizza";
 */
 
-var cupsOfFlour = 1;
+var cupsOfFlour = 3;
 var hasSauce = true;
 
 // Write your conditional here
-
+if ((cupsOfFlour <= 1) && (hasSauce == false)) {
+  console.log("I cannot make pizza");
+} else if ((cupsOfFlour <= 1) && (hasSauce == true)) {
+  console.log("I cannot make pizza");
+} else if ((cupsOfFlour >=2) && (hasSauce == true)) {
+    console.log("I can make pizza");
+} else {
+  console.log("Im hungry")
+}
```

Comment:
Now that you've learned more about logical operators...how would you change this if/else statement to process the situation where you have enough flour, but don't have sauce? Check this out on your own, no changes needed.